prettier.config.js

```diff
@@ -2,5 +2,7 @@ module.exports = {
   bracketSpacing: false,
   jsxBracketSameLine: true,
   singleQuote: true,
-  trailingComma: "all"
-};
+  trailingComma: 'all',
+  semi: false,
+  printWidth: 110,
+}
```

Comment:
Just curious, since we're changing the repo formatting as part of this CLI PR, is this the standard that we want for all blitzjs repos (website repo included)?

Reply:
I added this. I'm willing to flex on the printWidth, but not the `semi` 😄

Reply:
Oh man, that's it I'm out :door: J/k, I can ignore the lack of semi :laughing: And I don't have an opinion on the print width, I've always gone with the default based on their recommendation https://prettier.io/docs/en/options.html. Mostly was just curious so we can keep the website repo in sync.